### ISSUE TITLE
Feature Request: Add support for keyboard shortcuts.

### ISSUE THREAD

**Issue by iamsid** [EsperoTech/yaade#153]
Being a tool for developers, shortcuts are a must. Was a bit disappointed to not see shortcut support here.
Please add shortcuts to the interface like pressing enter/tabs while in params view to add another item.. ctrl+enter to run the API.. and any others that might work on a browser.

Postman shortcuts for reference: https://blog.postman.com/speeding-up-workflows-with-keyboard-shortcuts/

**Comment by jonrosner** [comment]
yes this is planned. I want to add a command-bar instead of keyboard shortcuts though, maybe similar to how vim works.
